addons/website/tests/test_snippets.py

```diff
@@ -77,3 +77,13 @@ def test_10_parallax(self):
 
     def test_11_snippet_images_wall(self):
         self.start_tour('/', 'snippet_images_wall', login='admin')
+
+    def test_12_snippet_popup_and_scrollbar(self):
+        website = self.env.ref('website.default_website')
+        website.cookies_bar = True
+        self.start_tour(self.env['website'].get_client_action_url('/'), 'snippet_popup_and_scrollbar', login='admin')
+
+    def test_12_snippet_popup_and_animations(self):
+        website = self.env.ref('website.default_website')
+        website.cookies_bar = True
```

Comment:
Actually since you repeat those steps, just merge both methods and do it once, then run the 2 tour back to back.
It's even better perf wise IMO (hot state?)